frontend/src/app/admin/indexing/status/CCPairIndexingStatusTable.tsx

```diff
@@ -200,8 +200,8 @@ function ConnectorRow({
         router.push(`/admin/connector/${ccPairsIndexingStatus.cc_pair_id}`);
       }}
     >
-      <TableCell className="">
-        <p className="lg:w-[200px] xl:w-[400px] inline-block ellipsis truncate">
+      <TableCell className="!w-[300px]">
```

Comment:
Typo error, remove "!"

Reply:
same as other

Reply:
Sorry for my mistake since I haven't used that before. But anyway, I think we should not use any "!important" since it will make it hard for us to debug if any error occur in the future

Reply:
This just means this specific element should have w-[300px] regardless of its parent style (if any). This is important since overflow inside text might change the table width.

Reply:
Okay. I will check this later, LGTM.